ia/src/ml_agent.py

```
@@ -0,0 +1,155 @@
+
+  while current_index < total_cells:
+    line = distance * 2 + 1
+    if current_index + line > index:
+      x = index - current_index - distance
+      y = distance
+      return (x, y)
+    current_index += line
+    distance += 1
+  return (0, 0)
+
+def search_closest_food(client):
```

Ruff (0.11.9)

Missing return type annotation for public function `search_closest_food`

(ANN201)